Improved support for MS Sans Serif

farmerbb opened the pull request:
This PR improves support for MS Sans Serif by including the font file in the repo, and also including a fonts.conf file to disable antialiasing for this specific font without requiring the user to disable antialiasing system-wide.

EMH-Mark-I commented:
I'm sure this would cause a licensing conflict to include fonts like that? Sure, we have the old trademarked MS Windows logo for the boot splash theme and then there's the icon theme too! I'm just afraid to push our luck.

I intended to leave it up to the reader to source the font themselves from a Windows installation instead of including it with the theme on GitHub.

farmerbb commented:
Totally understandable.  I've removed the font file and updated INSTALL.MD to bring back the information about sourcing the font from a Windows install.

EMH-Mark-I commented:
Thanks!
